polyglot/piranha/test-resources/java/feature_flag_system_1/treated/input/XPFlagCleanerPositiveCases.java

```diff
@@ -34,6 +34,16 @@ enum AnotherTestExperimentName {
     SOME_OTHER_FLAG
   }
 
+  public enum Something implements ExperimentName {
+    /** Show menu */
+    SHOW_MENU,
+  
+    /** Foobar */
```

Comment:
Shouldn't the treated **input** still have an XP flag enum here?

Reply:
Done. Thanks good catch.